options.go

```diff
@@ -15,13 +15,23 @@ var (
 		contentEncoding: &encodingOpt{
 			level: DefaultCompression,
 		},
+		retryOnStatus: map[int]bool{
```

Comment:
Can we document this in the README?

Reply:
I mean, if I'm correct, our client support it now, right?

Reply:
we can have section in the Getting Started section of the README, like "Customize client"